src/utility/svd.jl

```diff
@@ -131,7 +149,9 @@ function ChainRulesCore.rrule(
             n_vals = length(Sdc)
             lvecs = Vector{Vector{scalartype(t)}}(eachcol(Uc))
             rvecs = Vector{Vector{scalartype(t)}}(eachcol(Vc'))
-            minimal_info = KrylovKit.ConvergenceInfo(length(Sdc), nothing, nothing, -1, -1)  # Just supply converged to SVD pullback
+            minimal_info = KrylovKit.ConvergenceInfo(length(Sdc), nothing, nothing, -1, -1)  # Only num. converged is used
+            minimal_alg = GKL(; tol=1e-6)  # Only tolerance is used for gauge sensitivity
```

Comment:
is this algorithm not part of the `SVDAdjoint` struct?

Reply:
In the case that `SVDAdjoint.fwd_alg isa IterSVD` yes, but now also algorithms with `SVDAdjoint.fwd_alg isa FixedSVD` have to use that rrule. However, they only carry U, S and V. That means in that case you don't have access to the original algorithm that produced the U, S and V (also not necessarily `GKL`).